TESTING-INSTRUCTIONS.md

```diff
@@ -3,6 +3,13 @@ Testing instructions
 
 ## Unreleased
 
+### Add gross sales column to CSV export #6567
+
+1. Create more than 25 revenues
```

Comment:
I think the testing steps need a small adjustment here:

```
1. Navigate to Analytics -> Revenue
2. Adjust the date filter so that more than 25 rows are visible
4. Click "Download"
5. Click the download link in the email
6. See gross sales column
```

Reply:
Thank you for the feedback! The changes have been pushed.